code/_global_vars/lists/names.dm

```diff
@@ -0,0 +1,17 @@
+/*GLOBAL_LIST_INIT(ai_names, world.file2list("config/names/ai.txt"))
+GLOBAL_LIST_INIT(wizard_first, world.file2list("config/names/wizardfirst.txt"))
+GLOBAL_LIST_INIT(wizard_second, world.file2list("config/names/wizardsecond.txt"))
+GLOBAL_LIST_INIT(ninja_titles, world.file2list("config/names/ninjatitle.txt"))
+GLOBAL_LIST_INIT(ninja_names, world.file2list("config/names/ninjaname.txt"))
+GLOBAL_LIST_INIT(commando_names, world.file2list("config/names/death_commando.txt"))
+GLOBAL_LIST_INIT(first_names_male, world.file2list("config/names/first_male.txt"))
+GLOBAL_LIST_INIT(first_names_female, world.file2list("config/names/first_female.txt"))*/
+GLOBAL_LIST_INIT(last_names, world.file2list("config/names/last.txt"))
+/*GLOBAL_LIST_INIT(clown_names, world.file2list("config/names/clown.txt"))
+
+
+GLOBAL_LIST_INIT(verbs, world.file2list("config/names/verbs.txt"))
+GLOBAL_LIST_INIT(adjectives, world.file2list("config/names/adjectives.txt"))
+//loaded on startup because of "
+//would include in rsc if ' was used
+*/
```

Comment:
That raises many questions.
Why added so much commented code? 
Why not it replace original definations?
Why that add (but comment out) removed by us vars?

Reply:
didn't paid attention